src/prebid.js

```diff
@@ -171,6 +171,36 @@ function getWinningBidTargeting() {
   return winners;
 }
 
+exports.getWinningBidTargeting = function() {
```

Comment:
It seems you are exporting functions for the sole purpose of making them available to test. We've been following the general principle that we should test the public API of a module, that is the `exports` and ES2015 style named `export`s, but not the internal implementation of a module. Test coverage should show that comprehensive testing of the public API also covers internal implementation, or where it does not it indicates insufficient tests or cruft in the code. In any event, the export of functions in order to test should be avoided as it can be misleading as to the intent of the module.